src/Lexer.js

```diff
@@ -80,6 +80,8 @@ var tokenRegex = new RegExp(
     "([ \r\n\t]+)|" +                                 // whitespace
     "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" +  // single codepoint
     "|[\uD800-\uDBFF][\uDC00-\uDFFF]" +               // surrogate pair
+    "|\\\\verb\\*([^\0]).*?(?:\\3|\n|$)" +            // \verb*
+    "|\\\\verb([^*a-zA-Z]).*?(?:\\4|\n|$)" +          // \verb unstarred
```

Comment:
Detecting unmatched delimiters was my goal, but that approach works too.  Changed.